src/components/Articles/ListOfArticles/ListOfArticles.js

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { getAllArticles } from '../../../actions';
+import 'dotenv/config';
+import placeholder from '../../../assets/images/placeholder.png';
+import TimeStamp from '../../../helpers/TimeStamp';
+import './listOfArticles.scss';
+
+const { REACT_APP_IMAGE_BASE_URL } = process.env;
+export class ListsOfArticles extends Component {
+  state = { imageRectangle: 'w_400,ar_16:9,c_fill,g_auto,e_sharpen' };
+
+  async fetAllArticles() {
+    await this.props.getAllArticles();
+  }
+
+  componentWillMount() {
+    this.fetAllArticles();
+  }
+
+  render() {
+    const { articles } = this.props;
+    const { imageRectangle } = this.state;
+    return (
+      <div className="row" id="articleCard">
+        {(articles || []).map((article, key) => (
+          <div key={key}>
+            <div className="card">
+              <div className="small-screen-4 medium-screen-1 large-screen-1">
+                <div className="image">
+                  <Link to={`/article/${article.slug}`}>
+                    <img
```

Comment:
It would be better if you use the Img component to display the image

Reply:
Thanks man, got it